packages/server/lib/browsers/firefox.ts

```diff
@@ -281,6 +282,8 @@ const defaultPreferences = {
   // allow getUserMedia APIs on insecure domains
   'media.devices.insecure.enabled':	true,
   'media.getusermedia.insecure.enabled': true,
+
+  'marionette.log.level': process.env.DEBUG ? 'Debug' : undefined,
```

Comment:
how verbose is this going to be?

Reply:
after running the 19 example spec files, it looks like the extra logs tagged with Marionette amount to 309 lines: https://github.com/flotwig/test-cypress-action/runs/679150446

without this log level, there are 38 lines: https://github.com/flotwig/test-cypress-action/runs/678939258

do you think it should be removed? i added it because it helped me see the logs from when the TCP server is started: https://searchfox.org/mozilla-central/rev/3ce874dc2703831af3e5ef3a1d216ffd08057fa5/testing/marionette/components/marionette.js#508

Reply:
note, this will still only log out if the user has enabled `cypress:launcher` logs, since it prints to Firefox's stderr. we don't pipe browser stdio to Cypress stdio directly

Reply:
I think its good